src/avtk/clipselector.cxx

```diff
@@ -232,7 +277,49 @@ void ClipSelector::resize(int X, int Y, int W, int H)
 	redraw();
 }
 
+void setRecordBarsCb(Fl_Widget *w, void* data)
+{
+    if(!w || !data)
+        return;
+
+	ClipSelector *track = (ClipSelector*)w;
+	long bars = (long)data;
+	if(bars == -2){        
+		const char* answer = fl_input("Enter a custom number: ");
+        bars = atoi(answer);
+        if(!answer || bars <= 0|| bars > MAX_BARS) {
+            bars = -1;
+            fl_message("Invalid Input. You may enter a number smaller than %i!", MAX_BARS);
+        } else
+			bars = atoi(answer);
```

Comment:
Else here isn't required, given the `atoi()` above will always already have executed